apps/example-app/src/app/app.module.ts

```diff
@@ -28,8 +28,8 @@ function reducerItems() {
 }
 
 @NgModule({
-  declarations: [
-    AppComponent,
+  declarations: [AppComponent],
```

Comment:
question: can the `AppComponent` not be converted to a standalone component?

Reply:
That's a valid question @LayZeeDK 
Ideally, we should also convert this, or just remove it since this project is mainly used for tests (and not to serve it).
What do you think @markgoho ?